test/org/sagebionetworks/bridge/services/StudyServiceTest.java

```diff
@@ -361,6 +363,7 @@ private void assertStudyDefaults(Study study) {
     private void assertStudyChanged(Study study) {
         assertFalse(study.isStudyIdExcludedInExport());
         assertFalse(study.isEmailVerificationEnabled());
+        assertTrue(study.isAutoVerificationPhoneSuppressed());
```

Comment:
Minor: It's probably not necessary to have this in assertStudyChanged(), setStudyDefaults(), or changeStudyDefaults().